app/assets/js/script.js

```diff
@@ -4,6 +4,11 @@
   var $win = $(window);
   var $doc = $(document);
 
+  // For assetgraph to pick up strings as urls
```

Comment:
Assetgraph uses static analysis to figure out what files need to go in the build. However, non-module loader file loads, like `$.getJSON('/blacklist.json')` is not something it can recognize as a URL, since it's just a string. So in order for assetgraph to pick it up as a url that should be included in the graph I wrap it in `GETSTATICURL()`.

An alternative way to get assetgraph to pick up `/blacklist.json` in the build would be to name that file explicitly as an initial asset in the `reduce` section of the Gruntfile. This will ensure the file is part of the build, but it won't get revved, since initial assets are treated as files that should be externally linkable.